src/tools/tidy/src/deps.rs

```diff
@@ -445,7 +446,7 @@ const PERMITTED_RUSTC_DEPENDENCIES: &[&str] = &[
     "windows_x86_64_gnu",
     "windows_x86_64_gnullvm",
     "windows_x86_64_msvc",
-    "wit-bindgen-rt@0.33.0", // via wasi
+    "wit-bindgen-rt",
```

Comment:
The version pinning here is intentional (see https://github.com/rust-lang/rust/pull/136395#issuecomment-2692769062) -- can you retain a potentially new version pin and add a comment here linking to why?